servers/rendering/renderer_rd/storage_rd/light_storage.cpp

```diff
@@ -1360,6 +1364,9 @@ void LightStorage::reflection_probe_release_atlas_index(RID p_instance) {
 	ERR_FAIL_COND(!atlas);
 	ERR_FAIL_INDEX(rpi->atlas_index, atlas->reflections.size());
 	atlas->reflections.write[rpi->atlas_index].owner = RID();
+
+	// TODO investigate if this is enough? shouldn't we be freeing our textures and framebuffers?
```

Comment:
I think there is a leak here, it doesn't seem to be freeing our allocated radiance buffers.

Reply:
No, the atlas should manage freeing the textures and framebuffers. It allocates them all at program start and then frees them all at the end. This is just for other resources taking ownership of a single layer of the texture array.